homeassistant/components/vevor_heater/__init__.py

```diff
@@ -0,0 +1,66 @@
+"""The Vevor Heater integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components import bluetooth
+from homeassistant.components.bluetooth import (
+    BluetoothCallbackMatcher,
+    BluetoothScanningMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import VevorHeaterUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Vevor Heater from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Please don't set anything in hass.data until after passing the first refresh successfully

Reply:
Done.